### ISSUE TITLE
ML surface model elevation predictions are returning too high.

### ISSUE THREAD

**Issue by CoryK8nn8dy** [RedRocksCommunityCollege/Elevation-Descent#2]
The ML surface model is predicting elevations for Lookout Mt between 3.9-4.2 million feet. Primary suspect is a normalization/de-normalization issue.

**Comment by CoryK8nn8dy** [comment]
ml_model.py's test_one() function was de-normalizing the results from keras.model.predict(), however, these results were already de-normalized when returned from predict(). This caused the exaggerated elevation values. Issue closed.
